### DISCUSSION TITLE
Widget not working (only displays a blank window)

### DISCUSSION THREAD

**Discussion by maggie-j-liu** [johnlindquist/kit#764] [Errors] [answered]
Hi! When I try running a script involving a widget (like the example `Corgi` script or the `widget-hello-world` script) the widget displayed is blank. I'm using version 1.11.1 on an Apple Silicon Mac.

<img width="592" alt="image" src="https://user-images.githubusercontent.com/63619830/166196201-48a7b861-5e00-48dd-b14f-8f0af27168fc.png">

Does anyone know why this is happening or if there's a way to debug the widget window?

**Comment by johnlindquist** [comment, marked as answer]
Ah, dangit, that's my bad, I was in the middle of refactoring/extracting the app-specific stuff from the Kit sdk.

A quick fix until the next build:

```bash
cd ~/.kit
./node/bin/npm i petite-vue
```

I'll try to get a new build out on Wednesday

**Reply by maggie-j-liu** [reply]
Thanks, that works!
